pyproject.toml

```
@@ -49,6 +49,7 @@ dependencies = [
     "ordered-set (==4.0.2)",
     "pandas (>=2.2.3)",
     "psycopg2-binary (==2.9.10)",
+    "pyarrow (>=18.0.0,<19.0.0)",
```

```suggestion
    "pyarrow (>=18.1.0,<19.0.0)",
```

Would recommend we don't allow people to go older than the version in pyproject.toml, note this will require a relock